tests/ui/UnreadIndicatorsTest.js

```diff
@@ -488,7 +488,7 @@ describe('Unread Indicators', () => {
                 const hintText = Localize.translateLocal('accessibilityHints.chatUserDisplayNames');
                 const displayNameTexts = screen.queryAllByLabelText(hintText);
                 expect(displayNameTexts).toHaveLength(1);
-                expect(lodashGet(displayNameTexts[0], ['props', 'style', 'fontWeight'])).toBe(fontWeightBold);
+                expect(lodashGet(displayNameTexts[0], ['props', 'style', 'fontWeight'])).toBe(FontUtils.fontWeight.bold);
```

Comment:
Could you add some QA steps that cover the font changes?

Reply:
I did. Is that not enough?

> Same as in Tests.
> 
> Additionally:
> 
> 1. Check that there are no console errors on all platforms
> 2. Check that fonts look as expected (no font changes)